.gitignore

```diff
@@ -37,6 +37,7 @@ GeoIPv6.dat
 *node_modules/
 */node_modules/
 */bower_components/
+yarn-error.log
```

Comment:
This could probably be a bit harsher: `*.log`. Unsure why anyone would want to keep log files in a GitHub repo?